pymc/data.py

```diff
@@ -148,6 +147,10 @@ def __str__(self):
         return "Minibatch"
 
 
+def has_random_ancestor(r):
+    return rvs_in_graph(r) != set()
```

Comment:
```suggestion
    return bool(rvs_in_graph(r))
```
But then you might as well skip the one liner and just do `not rvs_in_graph(r)` below?

Reply:
Done